src/hotspot/share/utilities/bitset.inline.hpp

```diff
@@ -62,11 +89,13 @@ inline CHeapBitMap** BitSet::BitMapFragmentTable::lookup(uintptr_t key) {
   return NULL;
 }
 
-inline BitMap::idx_t BitSet::addr_to_bit(uintptr_t addr) const {
+template<MEMFLAGS F>
+inline BitMap::idx_t BitSet<F>::addr_to_bit(uintptr_t addr) const {
   return (addr & _bitmap_granularity_mask) >> LogMinObjAlignmentInBytes;
```

Comment:
See my earlier comment. This is a runtime variable, would be nice to have this as compile time constant.